README.md

```diff
@@ -1,17 +1,21 @@
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/com.phikal.regex/)
+
 ReGeX is a Android game that's all about ["Regular Expressions"][regex].
 In different modes the user can try to solve tasks, which range from
-having to match only ceratin strings but not others, or extracting
+having to match only certain strings but not others, or extracting
 parts of a string with a regular expression, making it enjoyable (under
 a certain definition of pleasure) for beginners and people with more
 experience alike.
 
 More information can be found on the [project homepage][website],
 the app can be downloaded from [F-Droid][fdroid], and most of the
 development (including issues and suggestions) is to be found on
-[GitHub][github]. ReGeX is published under [GPL v3][license].
+[GitHub][github]. ReGeX is published under [GPLv3][license].
 
 [regex]: https://en.wikipedia.org/wiki/Regular_expression
-[website]: https://zge.us.to/proj/ReGeX/
-[fdroid]: https://en.wikipedia.org/wiki/Regular_expression
-[github]: https://github.com/phikal/ReGeX/
-[license]: ./LICENSE
+[website]: https://github.com/phikal/ReGeX/wiki
+[fdroid]: https://f-droid.org/packages/com.phikal.regex/
+[github]: https://github.com/phikal/ReGeX
+[license]: https://www.gnu.org/licenses/gpl-3.0.en.html
```

Comment:
What's the rationale behind this change? GitHub already links to the README, and when viewed locally, one doesn't have to open a browser to read the license.

Reply:
It's just me trying to make the links as correct as possible. It isn't really necessary. I'll change it back if you want.

Reply:
Markdown-wise, they are both equally valid as links. But thinking about it, I guess it's not a big deal, so you can also leave it as it is.